homer0/wootils
# v2.6.4: fix(shared/apiClient): use the 'content-length' header to validate a response size

### What does this PR do?

This is a follow up for #48: I used `size` to validate if the response had content, but for some reason, on Node with [`node-fetch`](http://yarnpkg.com/en/package/node-fetch), `size` is always `0`. I added another check that validates the response `content-length` in case it's available.

Now, if the response has no `content-length` nor `size`, it may be something wrong with the API :P.

Also, I took the opportunity to update all dependencies and change the ESLint plugin I was using and replace it with [my own](https://www.npmjs.com/package/eslint-plugin-homer0).

### How should it be tested manually?

Use the `apiClient` on Node, with `node-fetch`,  against any kind of endpoint; if it returns a valid `content-length`, it should parse it.

And of course...

```bash
yarn test
# or
npm test
```
